### ISSUE TITLE
Install and configure Search API Best bets

### ISSUE THREAD

**Issue by leeomara** [ytgov/yukon-ca#702]
In order to improve our search results, we want to be able to create and managed "pinned" or "best bet" results for specific queries.

I think the Search API Best bets module will allow us to do this
  https://www.drupal.org/project/search_api_best_bets

- [ ] install Search API best bets
- [ ] review and update permissions for access to Best bets (should be restricted to small group of users)
- [ ] catalogue the most prominent missing matches for each of the top 25 searches

**Comment by lgontard** [comment]
@leeomara Is this still a high priority?

**Comment by leeomara** [comment]
Nope, not a high priority. We might come back to this after evaluating our search performance position, but maybe not?
